atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -431,6 +432,8 @@ def render_modules_request(self):
         }
         if self.arches:
             request['arches'] = self.arches
+        if self.modular_tags:
+            request['modular_koji_tags'] = ' '.join(self.modular_tags)
```

Comment:
In the previous commit, `start_compose` docstring says it is supposed to receive a list of strings (ODCS code also seems to be expecting a list). Is there a reason for having a string here?

Reply:
I got confused. Fixed.